Manual.md

```diff
@@ -914,8 +911,8 @@ can be used to pass arguments during compilation. If your package does not make
 extensions consider using the `gem` build style instead.
 
 - `gem` For packages that are installed using gems from [RubyGems](https://rubygems.org/).
-The gem command can be overridden by `gem_cmd`. `archs` is set to `noarch` unconditionally
```

Comment:
Not removed from code.

Reply:
I swear I did this....

Reply:
(in next push)

Reply:
Actually this will come in the first batch set with revbumps of `build_style=gem` packages.